src/main/java/com/adobe/commerce/cif/model/category/Category.java

```diff
@@ -24,7 +24,7 @@
     @ApiModelProperty(value = "The internal unique ID of the category in the commerce backend system.", required = true)
     protected String id;
 
-    @ApiModelProperty(value = "The name of the category.")
+    @ApiModelProperty(value = "The name of the category.", required = true)
```

Comment:
I'd not make this required, there are many cases where we cannot easily get that. For example for `/products` endpoints and for the parent/sub categories of categories in `/categories` endpoints.